### ISSUE TITLE
Include Sympa commands doc

### ISSUE THREAD

**Issue by ldidry** [sympa-community/sympa-community.github.io#21]
Include https://www.sympa.org/manual/sympa-commands in new site

**Comment by ikedas** [comment]
Hi @ldidry,

Contents of www.sympa.org are no longer maintained.  Part of contents moved from there to this site got permission of original owners to clarify license (see also commit log).

Recent version of command helps are also there:
  - default/web_tt2/help_commands.tt2
  - default/mail_tt2/helpfile.tt2

I’m planning to move those help content to this site (you may find planned link in top page).

**Comment by ldidry** [comment]
> Recent version of command helps are also there

Yes, I just wanted to link a readable page instead of tt2 templates.

> I’m planning to move those help content to this site

I saw the "coming soon" links, but by opening this issue (and assigning it to myself), I wanted to say "I'm planning to do this part so you can focus on other parts".

Just for information: I know that I may not create the perfect page (either because I can write things that not fully integrate with the theme you choose, or you may want to group commands by whatever topic, I don't know), but at least I think it would be a good start to have the old doc in markdown and fixed (the command page is slightly messed up, not the content, but display).
